manifests/profile/neutron/router.pp

```diff
@@ -12,14 +12,17 @@
 
   ### Router service installation
   class { '::neutron::agents::l3':
-    debug                   => $::openstack::config::debug,
-    external_network_bridge => 'brex',
-    enabled                 => true,
+    debug                    => $::openstack::config::debug,
+    external_network_bridge  => 'brex',
+    enabled                  => true,
+    agent_mode               => 'dvr_snat',
+    router_delete_namespaces => 'True',
```

Comment:
should be Boolean true-false

reference:
http://puppet-lint.com/checks/quoted_booleans/